cms/djangoapps/contentstore/views/permissions.py

```diff
@@ -0,0 +1,22 @@
+"""
+Custom permissions for the content store views.
+"""
+
+from rest_framework.permissions import BasePermission
+
+from common.djangoapps.student.auth import has_studio_write_access
+from openedx.core.lib.api.view_utils import validate_course_key
+
+
+class HasStudioWriteAccess(BasePermission):
```

Comment:
so same permission exists so why not use that one ?

Reply:
The key was different :/  We are use `course_key_string` and that one uses `course_id` as kwarg.

Reply:
in that we can go withour own.